inst/tests/tests.Rraw

```diff
@@ -15671,6 +15671,15 @@ if (test_bit64) {
   test(2077.06, int64_int32_match(d[, sum(i32, na.rm=TRUE), g], d[, sum(i64, na.rm=TRUE), g]))
 }
 
+# #1926 -- unlock .SD for bmerge in j
+DT = data.table(id=1:2, v=3:4)
+DT2 = data.table(id=1, x=5)
+## DT2.i is numeric so bmerge does coercion with set()
+test(2078.1, DT[id == 1, DT2[.SD, on="id"]], data.table(id=1L, x=5, v=3L))
```

Comment:
is there a tests that verify that `i` data.table was not altered after join?

Reply:
I would assume so but it's a good idea to add one explicitly here.

Reply:
@MichaelChirico still missing this test, isn't it?